### PR TITLE
Show kano-nav on the /start view

### PR DESCRIPTION
**Pull request by abridger**
Trello card: https://trello.com/c/n4koEAte/1697-2-implement-global-navigation-on-kano-me-and-start-page

### CODE REVIEW

File: src/elements/kw-view-start/kw-view-start.html [hunk 1]
```diff
@@ -123,7 +97,7 @@
                 border: none;
                 border-radius: 30px;
                 padding: 15px 35px;
-                font-family: 
+                font-family:
```
Comment (paulvarache) [inline src/elements/kw-view-start/kw-view-start.html]: WUT?

Comment (abridger) [reply src/elements/kw-view-start/kw-view-start.html]: Bizarre. Sorry about that. Will remove now!

**Review by vikdoro — COMMENTED** [review]
Sorry, I named the listener function on line 265 and 281 (that sets up the simplePage) too specific to the navbar. Maybe now it's a good chance to rename it to something more generic?

**Comment by abridger** [comment]
Good point! Sorry, I wasn't very thorough. I'll update that now.

**Comment by vikdoro** [comment]
Perfect, thanks 👍

**Pull request merged**